package.json

```
@@ -24,7 +24,7 @@
 		"svelte-check": "^4.0.0",
 		"tailwindcss": "^3.4.9",
 		"typescript": "^5.0.0",
-		"vite": "^5.4.11"
+		"vite": "^6.1.0"
```

incompatible with other packages!
```suggestion
		"vite": "^5.4.11"
```